### ISSUE TITLE
panrgp command not working

### ISSUE THREAD

**Issue by lsemenec** [labgem/PPanGGOLiN#40]
Hello,

I've installed the latest version 1.1.108 and ran the workflow command which had no issues. However, when trying to run the panrgp command as described:
ppanggolin panrgp --fasta genomes_list.txt
I get the error message:
ppanggolin: error: argument : invalid choice: 'panRGP' (choose from 'annotate', 'cluster', 'graph', 'partition', 'rarefaction', 'workflow', 'draw', 'write', 'align', 'info')

I even went to my pangenome files generated from the workflow command and tried:
ppanggolin rgp -p pangenome.h5 
 
but got the same error message.
ppanggolin: error: argument : invalid choice: 'rgp' (choose from 'annotate', 'cluster', 'graph', 'partition', 'rarefaction', 'workflow', 'draw', 'write', 'align', 'info')

I also tried running the panRGP.py script in the workflow dir but that didn't work either. Can you please let me know how I can go about running the panRGP command?

Many thanks!

**Comment by ggautreau** [comment]
Hi @Isemenec,

What is your output of the command `ppanggolin -v` ?

Thx.

**Comment by axbazin** [comment]
Hello @isemenec, this is very surprising, especially if this is the 1.1.108 version. The CI worked fine and I just tested a clean install just to be sure.

Sounds like you installed through git, since 1.1.108 isn't available on conda, so I did the same and both commands work on my end with the current master branch.

It sounds like you have an older version of ppanggolin installed though (since originally both of those commands were not available). The output of `ppanggolin -v` could definitely help tracking the problem down.

If it's something under 1.1.72, you will need to do reinstall ppanggolin.
